tests/components/arcticspa/__init__.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for the Arctic Spa integration."""
+
+from unittest.mock import AsyncMock, MagicMock, Mock
+
+from pyarcticspas import Spa
+
+from homeassistant.components.arcticspa import CONF_API_KEY
+from homeassistant.components.arcticspa.const import DOMAIN
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+CONF_API_KEY_VALUE = "mockapikey"
+
+
+def _mock_spa() -> Spa:
+    spa = MagicMock(auto_spec=Spa, name="Mocked Arctic Spa Device")
+    spa.async_status = AsyncMock()
+    spa.status = Mock()
+    spa.async_set_lights = AsyncMock()
+    spa.set_lights = AsyncMock()
+    return spa
```

Comment:
Instead of making a function for this, maybe just create a mock fixture. Please checkout Epion tests

Reply:
Epion tests were a great help, thanks! fixture created.